procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "3dbf3c0c",
```

Comment:
<p>Why is this here? If required, please add a line stating the reasoning.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
<p>Deleted since the dome motion is tested in previous cell with </p><p><code>await atcs.rem.atdome.cmd_moveAzimuth.set_start(azimuth=dome_az.azimuthPosition+d_az)</code></p><p> </p>